source/chapters/user_interface.rst

```diff
@@ -591,12 +602,15 @@ control rate changes also from your computer's keyboard, see the chapter
   +0.0%, which indicates no change.
 
 **Temporary Pitch/Rate buttons (Nudge)**
-  Pushing the minus and plus buttons is like nudging the metal edge of
+  Pushing the diagonal arrow buttons is like nudging the metal edge of
```

Comment:
just an idea:
what's your opinion on using these characters? like the `+` `-` signs below
🡴 🡶 or  🡼 🡾

Reply:
![image](https://user-images.githubusercontent.com/28389950/112435186-7800ff80-8d55-11eb-9785-c88f9e886185.png)

I don't understand these last 2 symbols .... but the + and - buttons look good :) We should should def use those instead.

Reply:
leaving this open for reference